src/pages/General/helper.js

```diff
@@ -45,4 +45,5 @@ export const createInitialValues = prevGeneralDetails => ({
     encryptDB: prevGeneralDetails.encryptDB || defaultEncryptDB,
     allowScreenCapture:
         prevGeneralDetails.allowScreenCapture || defaultShareScreen,
+    messageOfTheDay: prevGeneralDetails.messageOfTheDay,
```

Comment:
Hey, 

It would be nice to have a default value just like the rest of the `prevGeneralDetails` settings. Something like this `prevGeneralDetails.messageOfTheDay || ''`.

Thanks!